generic/tests/linux_stress.py

```diff
@@ -0,0 +1,203 @@
+import time
+import logging
+import aexpect
+
+from avocado.utils import download
+
+from virttest import data_dir
+from virttest import utils_misc
+from virttest import error_context
+
+
+class Stress(object):
+    """
+    Stress function for linux guest
```

Comment:
/s/function/class

Reply:
@huangyum Here function I intend to mean utility, not that function(), It's truly confusing word, I will change it to utility. Thanks.